docs/en/users/05_Configuration.md

```diff
@@ -13,16 +13,23 @@ Available languages are: cz, de, en, es, fr, he, it, ko, nl, oc, pt-br, ru, tr,
 
 ## Theme
 
-There’s no accounting for tastes, which is why FreshRSS offers eight official themes:
-
-* *Blue Lagoon* by **Mister aiR**
-* *Dark* by **AD**
-* *Flat design* by **Marien Fressinaud**
-* *Origine* by **Marien Fressinaud**
-* *Origine-compact* by **Kevin Papst**
-* *Pafat* by **Plopoyop**
-* *Screwdriver* by **Mister aiR**
-* *Swage* by **Patrick Crandol**
+There’s no accounting for tastes, which is why FreshRSS offers 13 official themes:
+
+| Theme       | designed by                                   | Notes                                                         |
```

Comment:
Let's do the same for https://github.com/FreshRSS/FreshRSS/blob/edge/docs/fr/users/05_Configuration.md
I can help with the translation:

| Thème | Auteur | Notes |